src/node_options.cc

```diff
@@ -148,6 +148,11 @@ void EnvironmentOptions::CheckOptions(std::vector<std::string>* errors) {
     errors->push_back("invalid value for --unhandled-rejections");
   }
 
+  if (tls_min_v1_3 && tls_max_v1_2) {
```

Comment:
`node --tls-min-v1.3 --tls-max-v1.2 --tls-max-v1.3` currently does the right thing (sets min and max to 1.3) but this change makes it an error.

I'm okay with that, just pointing it out in case it's something we want to preserve. Changing the logic to `tls_min_v1_3 && tls_max_v1_2 && !tls_max_v1_3` would do that.

Reply:
I would keep the check as it is. Having multiple flags that partially contradict each other could confuse people.